app/views/reports/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :heading do %>
+  <h1><%= t ".title_html", :link => link_to(reportable_title(@report.issue.reportable), reportable_url(@report.issue.reportable)) %></h1>
+<% end %>
+
+<div class="report-disclaimer">
+  <%= t('issues.new.disclaimer.intro') %>
+  <ul>
+    <li> <%= t('issues.new.disclaimer.not_just_mistake') %> </li>
+    <li> <%= t('issues.new.disclaimer.unable_to_fix') %> </li>
+    <li> <%= t('issues.new.disclaimer.resolve_with_user') %> </li>
+  </ul>
+</div>
+
+<%= form_for(@report) do |f| %>
+  <%= f.error_messages %>
+  <fieldset>
+    <%= f.fields_for @report.issue do |issue_form| %>
+      <%= issue_form.hidden_field :reportable_id %>
+      <%= issue_form.hidden_field :reportable_type %>
+    <% end %>
+
+    <div class='form-row'>
+      <p><%= t('issues.new.select') %></p>
+      <ul>
+      <% Report.categories_for(@report.issue.reportable).each do |c| %>
+        <li>
+          <%= radio_button :report, :category, c %>
+          <%= label_tag "report_category_#{c}", t("reports.categories.#{@report.issue.reportable.class.name}.#{c}") %> <br/>
```

Comment:
I'm inclined to say we should run `underscore` on the class name here to make the locale keys more typical?